tests/congruence_tests/test_complex_filters.py

```
@@ -0,0 +1,230 @@
+    )
+
+    remote_result, _next_page = remote_client.scroll(
+        collection_name=COLLECTION_NAME,
+        scroll_filter=filter_,
+        limit=100,
+        with_payload=True,
+    )
+
+    # assert len(local_result) == len(remote_result)
+
+    for local, remote in zip(local_result, remote_result):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)